library/hash_info.c

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Reminder: the changelog will need to mention the new module, to inform people who use custom build scripts.